tests/integration/tier1/remove-excluded-pkgs/test_remove_excluded_pkgs_epel7.py

```diff
@@ -0,0 +1,30 @@
+from envparse import env
+
+
+def test_remove_excluded_pkgs_centos8(shell, convert2rhel):
+    """Ensure c2r removes pkgs, which specified as excluded_pkgs in config."""
+
+    excluded_pkg_1 = "gnome-documents-libs"
```

Comment:
Is this the only difference from `test_remove_excluded_pkgs_epel7.py`? If yes, can we have just one test but parametrized?

Reply:
See https://tmt.readthedocs.io/en/stable/spec/tests.html#environment

Reply:
@r0x0d was the removal of integration test intended?

Reply:
@danmyway mentioned today he still works on the improving the integration test.

Reply:
Test updated